docs/README.md

```diff
@@ -409,3 +448,15 @@ The format is: `${vendor}-${appName}-${appMajor}`
 ![new-configuration-test](https://user-images.githubusercontent.com/27698855/116406935-94d29c00-a807-11eb-80c6-190e8fd6d373.png)
 
 8. Configure a payment condition with your newly created connector and wait 10 minutes to appear on checkout!
+
+<br />
+
+## Making your connector available to everyone
+
+The publication process is made via app store, more info on how to do that here: 
+
+[Submitting your app in the VTEX app store](https://developers.vtex.com/vtex-developer-docs/docs/vtex-io-documentation-submitting-your-app-in-the-vtex-app-store)
+
+After the homolog step is complete, your app needs to be installed in the `gateewayio` account so that it will appear in the affiliations list. If an account tries to create an affiliation with your connector but does not have it installed, a warning will appear in the admin as such:
+
+![affiliation-warning](https://user-images.githubusercontent.com/27698855/117978635-7a6ce800-b308-11eb-9690-064892af30d6.png)
```

Comment:
I don't know if we need this here since the app can be installed regardless it was installed or not on `gatewayio` account.

Install an app on `gatewayio` account is a manual step made by us, so it makes no difference if we explain how it works under the hood,

Reply:
This was resolved by the changes you suggested, right?